src/sfctl/tests/mock_server.py

```diff
@@ -0,0 +1,69 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -----------------------------------------------------------------------------
+
+"""A mock server for testing purposes."""
+
+from http.server import (BaseHTTPRequestHandler, HTTPServer)
+from threading import Thread
+import socket
+import requests
+
+class MockServer(BaseHTTPRequestHandler):
+    """ Overrides the following methods in BaseHTTPRequestHandler """
+
+     # pylint: disable=E1101
```

Comment:
Don't use the number format identifiers, use the strings like `invalid-name` or `line-too-long`

Reply:
Updated